src/config.ts

```diff
@@ -32,15 +32,36 @@ export interface Config {
 const DEFAULTS: Config = {
   output: 'dist',
   pluginsDirectory: path.join(os.homedir(), '.cdk8s', 'plugins'),
+  imports: ['k8s'],
 };
 
-export function readConfigSync(): Config {
+export function readConfigSync(filePath?: string): Config {
   let config: Config = DEFAULTS;
-  if (fs.existsSync(CONFIG_FILE)) {
+  const fullFilePath = filePath ? path.join(filePath, CONFIG_FILE) : CONFIG_FILE;
+  if (fs.existsSync(fullFilePath)) {
+    const parsedYaml: Config = yaml.parse(fs.readFileSync(fullFilePath, 'utf-8'));
     config = {
       ...config,
-      ...yaml.parse(fs.readFileSync(CONFIG_FILE, 'utf-8')),
+      ...parsedYaml,
     };
   }
   return config;
+}
+
+export async function addImportToConfig(source: string, filePath?: string) {
```

Comment:
The function is called `addImport`, but it actually does more than that by persisting default values to the file.

Also, I actually don't think we should be writing anything else to the file. It creates a weird experience where the `cdk8s import` command suddenly adds unrelated things to the file. Think about it from a user perspective - would they understand why the file changed in such a way?

Lets simply read the current config, append the import to it, and write back. Then, the function is actually appropriately named :)